app/views/responsiblepeople/FitAndProperView.scala.html

```diff
@@ -48,6 +48,13 @@
 
         @inputYesNo(
             Radios(
+                fieldset = Some(Fieldset(
+                        legend = Some(Legend(
+                        content = Text(messages("responsiblepeople.fit_and_proper.legend", personName)),
+                        classes = "govuk-fieldset__legend--m",
+                        isPageHeading = true
```

Comment:
This should be false. Currently this is being rendered as another h1 even though visually it looks fine